apps/root/src/services/eventService.ts

```diff
@@ -17,6 +18,25 @@ export default class EventService {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     this.mixpanel = MixpanelLibray.init(process.env.MIXPANEL_TOKEN!, { api_host: MEAN_PROXY_PANEL_URL }, ' ');
     this.mixpanel.set_config({ persistence: 'localStorage', ignore_dnt: true });
+    if (process.env.HOTJAR_PAGE_ID) {
+      try {
+        Hotjar.init(Number(process.env.HOTJAR_PAGE_ID), 6);
+      } catch (error) {
+        console.error('Error initializing Hotjar', error);
+      }
+    }
+  }
+
+  identifyUser(userId?: string) {
+    if (!userId) {
+      return;
+    }
+    this.mixpanel.identify(userId);
```

Comment:
We are identifying user twice right? In login method from account service we would trigger it here and in `trackEvent` method

Reply:
There is no problem with this according to mixpanels docs :)